dcc-portal-ui/app/scripts/repository/js/repository.js

```diff
@@ -1131,4 +1132,11 @@
 
   });
 
+  // This is a workaroud required for Internationalization of 'Experimental&nbsp;Strategy'
+  module.filter('addEntity', function(){
+    return function(string){
+      return string.split(' ').join('\u00A0');
+    };
```

Comment:
When naming functions, filters, classes, think in terms of "would someone who only sees where this is used (without reading the function definition, or the documentation) be able to understand what it does? (or at least be able to make a decent guess)"

If you were to ask someone what they would assume a filter called `addEntity` does, I don't think they'd be able to guess that it  "replaces spaces with non breaking white space characters". 

Also not a fan of creating functions that can only do very specific things, unless it's used so frequently where it's worth it to create a partially bound variant of the function

In this instance, a `replace` filter would be appropriate

Usage would be something like
```html
{{'Experimental Strategy' | translate | replace: ' ':'\u00A0'}}
```

This makes it very clear what is happening, and the filter is also flexible enough to be useful in other places

In our case, since we already have lodash, we should be able to do this:

```js
module.filter('replace', function(){
  return _.replace;
});
```